src/components/DiseasesForProteinCardContainer.tsx

```diff
@@ -1,40 +1,32 @@
-import React, { Fragment, FunctionComponent } from "react";
-import { RouteComponentProps } from "react-router";
+import React, { FunctionComponent } from "react";
+import { RouteComponentProps, useParams } from "react-router";
 import { v1 } from "uuid";
-import useApi from "./UseApi";
+import useApi from "./hooks/UseApi";
 import DiseaseCard, { DiseaseData } from "./cards/DiseaseCard";
-import PageTemplate from "../PageTemplate";
+import PageTemplate from "../layout/PageTemplate";
 import { Context } from "../types/context";
-import PageContainer from "../PageContainer";
-import ProteinContainer from "./ProteinContainer";
 import { baseUrl } from "../config";
 
 const DiseasesForProteinCardContainer: FunctionComponent<RouteComponentProps<
```

Comment:
```suggestion
const DiseasesForProteinCardContainer: FunctionComponent
```

Reply:
Not using anything injected with a `with...` wrapper component anymore, so no need for that